custom_components/berlin_transport/sensor.py

```diff
@@ -38,42 +43,53 @@
 _LOGGER = logging.getLogger(__name__)
 
 TRANSPORT_TYPES_SCHEMA = {
-    vol.Optional(CONF_TYPE_SUBURBAN, default=True): bool,
-    vol.Optional(CONF_TYPE_SUBWAY, default=True): bool,
-    vol.Optional(CONF_TYPE_TRAM, default=True): bool,
-    vol.Optional(CONF_TYPE_BUS, default=True): bool,
-    vol.Optional(CONF_TYPE_FERRY, default=True): bool,
-    vol.Optional(CONF_TYPE_EXPRESS, default=True): bool,
-    vol.Optional(CONF_TYPE_REGIONAL, default=True): bool,
+    vol.Optional(CONF_TYPE_SUBURBAN, default=True): cv.boolean,
+    vol.Optional(CONF_TYPE_SUBWAY, default=True): cv.boolean,
+    vol.Optional(CONF_TYPE_TRAM, default=True): cv.boolean,
+    vol.Optional(CONF_TYPE_BUS, default=True): cv.boolean,
+    vol.Optional(CONF_TYPE_FERRY, default=True): cv.boolean,
+    vol.Optional(CONF_TYPE_EXPRESS, default=True): cv.boolean,
+    vol.Optional(CONF_TYPE_REGIONAL, default=True): cv.boolean,
 }
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Optional(CONF_DEPARTURES): [
             {
-                vol.Required(CONF_DEPARTURES_NAME): str,
-                vol.Required(CONF_DEPARTURES_STOP_ID): int,
-                vol.Optional(CONF_DEPARTURES_DIRECTION): int,
-                vol.Optional(CONF_DEPARTURES_DURATION): int,
-                vol.Optional(CONF_DEPARTURES_WALKING_TIME, default=1): int,
-                vol.Optional(CONF_SHOW_API_LINE_COLORS, default=False): bool,
+                vol.Required(CONF_DEPARTURES_NAME): cv.string,
+                vol.Required(CONF_DEPARTURES_STOP_ID): cv.positive_int,
+                vol.Optional(CONF_DEPARTURES_DIRECTION): cv.string,
```

Comment:
It seems like the Direction is a string and not an int, so I fixed that here https://v6.vbb.transport.rest/api.html#get-stopsiddepartures